Source/Dafny/DafnyAst.cs

```diff
@@ -1900,6 +1900,10 @@ public enum AutoInitInfo { MaybeEmpty, Nonempty, CompilableValue }
       } else if (a.IsNumericBased()) {
         // Note, for meet, we choose not to step down to IntVarietiesSupertype or RealVarietiesSupertype
         return a.Equals(b) ? a : null;
+      } else if (a.IsBitVectorType) {
+        var aa = a.AsBitVectorType;
```

Comment:
Can you not do something like `if (a.IsBitVectorType) return Equals(a,b) ? a : null`

Reply:
Yes, that's better--and makes it look like the other cases.